docs/populating-configuration.md

```diff
@@ -0,0 +1,353 @@
+  might be useful to get the IAM user/s credentials
+
+## Setup Fleet Manager GCP configuration
+Fleet Manager interacts with GCP to provide the following functionalities:
+* To be able to create and manage Data Plane clusters in a specific GCP account
+  by passing the needed credentials to OpenShift Cluster Management
+
+If you need any of those functionalities keep reading. Otherwise, this section
+can be skipped.
+
+If you intend to configure/provision Data Planes in GCP then
+[GCP Service Account](https://cloud.google.com/iam/docs/service-accounts)
+JSON credentials need to be provided to Fleet Manager so it can deploy and
+provision Data Plane Clusters there.
+To create a GCP Service Account and its corresponding JSON credentials see:
+  * [Creating GCP Service Accounts](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating)
+  * [Creating GCP Service Account Keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating)
+
+  Additionally, the GCP Service Account has to meet the following
+  requirements:
+
+  * In case the Data Plane Clusters are to be provisioned through OCM then
+    the GCP Service Account has to be named `osd-ccs-admin`
+  * In case the Data Plane Clusters are to be provisioned through OCM then
+    the following GCP IAM Roles have to be granted to the GCP Service Account:
+    [Required GCP IAM roles](https://docs.openshift.com/container-platform/latest/installing/installing_gcp/installing-gcp-account.html#installation-gcp-permissions_installing-gcp-account).
+    See [Manage Service Account access](https://cloud.google.com/iam/docs/manage-access-service-accounts#single-role)
+    for details on how to do it
+
+In order to configure GCP Service Account JSON credentials for Fleet Manager
+retrieve them from GCP and configure them. To do so the following
+alternatives are available:
+* Copy the contents of the JSON credentials into the
+  `secrets/gcp.api-credentials` file
+* Run the  `gcp/setup/credentials` Makefile target providing the JSON
+  credentials content as a base64-encoded string in the `GCP_API_CREDENTIALS`
+  environment variable. To do so run:
+  ```bash
+  GCP_API_CREDENTIALS="<base64-encoded-gcp-serviceaccount-credentials>" make gcp/setup/credentials
+  ```
+
+Finally, make sure that `gcp` is listed as a supported Cloud Provider with at
+least one configured [GCP region](https://cloud.google.com/compute/docs/regions-zones)
+in Fleet Manager in the `config/provider-configuration.yaml` configuration
+file. See the documentation in that file for detail on the configuration
+schema.
+
+## Setup additional SSO configuration
+A SSO server is needed in Fleet Manager to enable some functionalities:
+* To enable communication between the Fleetshard operator (in the data plane) and the Fleet Manager
+* To configure the OpenShift cluster identity provider in Data Plane clusters
```

Comment:
Another purpose of it is to allow a successful Kafka instance provision. Each Kafka instance requires an SSO service account for the Canary deployment.

Reply:
:+1: but I understand that is not strictly required.
We have a flag EnableAuthenticationOnKafka that I understand controls that requirement

Reply:
Added